phpstan-baseline.neon

```diff
@@ -1,9 +1,9 @@
 parameters:
 	ignoreErrors:
 		-
-			message: "#^Method Vich\\\\UploaderBundle\\\\Adapter\\\\AdapterInterface\\:\\:recomputeChangeSet\\(\\) has no return type specified\\.$#"
+			message: "#^Call to an undefined method Doctrine\\\\Persistence\\\\ObjectManager\\:\\:getUnitOfWork\\(\\)\\.$#"
```

Comment:
This error is ignored because the object returned from `getObjectManager()` is an instance of `Doctrine\ODM\PHPCR\DocumentManagerInterface` (https://github.com/doctrine/phpcr-odm/blob/master/lib/Doctrine/ODM/PHPCR/DocumentManagerInterface.php) but we are not installing this package so phpstan can't analyze this.